orttraining/tools/ci_test/run_bert_perf_test.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import subprocess
+import sys
+import os
+
+SCRIPT_DIR = os.path.realpath(os.path.dirname(__file__))
+
+def parse_args():
+  parser = argparse.ArgumentParser(description="Runs BERT performance tests.")
+  parser.add_argument("--binary_dir", required=True,
+                      help="Path to the ORT binary directory.")
+  parser.add_argument("--training_data_root", required=True,
+                      help="Path to the training data root directory.")
+  parser.add_argument("--model_root", required=True,
+                      help="Path to the model root directory.")
+  return parser.parse_args()
+
+# using the same params from "GitHub Master Merge Schedule" in OneNotes
+def main():
+    args = parse_args()
+
+    matrix = { # enable mixed-precision, sequence length, max batch size
```

Comment:
the fourth parameter is not mentioned, i assume it's max predictions per sequence.
namedtuple could be useful here: https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
will check ...

Reply:
nice article, but I'd like to keep it simple by adding just the missing field description. The matrix is 2D, so adding named tuples would make the code looks a bit longer.